hedera-node/cli-clients/src/test/java/com.hedera.services.cli.sign.test/RecordStreamSignCommandTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.cli.sign.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.hedera.services.cli.sign.RecordStreamSignCommand;
+import java.io.File;
+import java.nio.file.Path;
+import java.security.KeyPair;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.util.Objects;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.Mock;
+
+class RecordStreamSignCommandTest {
+    @Mock
+    PublicKey publicKey;
+
+    @Mock
+    PrivateKey privateKey;
+
+    @TempDir
+    private File tmpDir;
+
+    @Mock
+    private RecordStreamSignCommand subject;
+
+    @BeforeEach
+    void setUp() {
+        subject = new RecordStreamSignCommand();
+    }
+
+    @Test
+    @DisplayName("Failure to generate signature file for file record stream")
+    void failureGenerateSignatureFileRecordStream() {
+        // given:
+        final var signatureFileDestination = Path.of("testPath");
+        final Path fileToSign;
+        fileToSign = signatureFileDestination;
+        final var keyPair = new KeyPair(publicKey, privateKey);
+        final var hapiVersion = "0.37.0-allowance-SNAPSHOT";
```

Comment:
This value is passed in on the command line with -hv 0.37.0-allowance-SNAPSHOT. Here for the unit test, since most files are signed with this version, I used this value. Also moved it's assignment to TestUtils.